doc/user-guide.md

```diff
@@ -278,33 +276,31 @@ We can now invoke the Write RPC on each `node` in the configuration:
   }
 ```
 
-While Gorums allows to call RPCs on single nodes as we did above, Gorums also provides a call type _quorum call_ that allows us to invoke a RPC on all nodes in a configuration with a single invocation, as we show in the next section.
+While Gorums allows us to call RPCs on individual nodes as we did above, Gorums also provides a call type _quorum call_ that allows us to invoke an RPC on all nodes in a configuration with a single invocation, as we show in the next section.
 
 ## Quorum Calls
 
-Instead of invoking an RPC explicitly on all nodes in a configuration, Gorums allows users to invoke the RPC as a quorum call on the configuration.
-If an RPC is invoked as a quorum call, Gorums will invoke the RPCs on all nodes in the configuration in parallel, and collect and process the replies.
+Instead of invoking an RPC explicitly on all nodes in a configuration, Gorums allows users to invoke a _quorum call_ via a method on the `Configuration` type.
+If an RPC is invoked as a quorum call, Gorums will invoke the RPCs on all nodes in parallel and collect and process the replies.
 
-For the Gorums plugin to generate quorum calls we have to specify the `quorumcall` option for our RPC methods in the proto file, as shown below:
+For the Gorums plugin to generate quorum calls we need to specify the `quorumcall` option for our RPC methods in the proto file, as shown below:
 
 ```protobuf
 service QCStorage {
```

Comment:
This requires `gorums.proto` to be imported, and that has not been mentioned at this point.